### DISCUSSION TITLE
Can't assign collaborator to Item. They don't show in dropdown "Suggested" list or when searching

### DISCUSSION THREAD

**Discussion by davidross2020** [community/community#38291] [Projects and Issues]
Hi all!

I have an issue with trying to assign a collaborator to a task in Projects. 

I have old tasks from before I invited the collaborator and new tasks to test if the old tasks were the issue. I click on Search, start typing their user name, and nothing shows up. Well "🚫 No Matches" appears. They do have write access to the Project and have been updating and editing tasks.

I did notice their's is the only icon that appears in the top right next to the Insights icon, and it seems to be dimmer/greyed out. They are the only non-admin collaborator in the project if that makes sense.

**Comment by ossanna16** [comment]
Hi there @davidross2020 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by NicholasHite** [comment]
Seeing the same issue

**Comment by rvsuper8** [comment]
Same issue here. New issue user shows up, old issue, user not in list to assign.

**Comment by KyanR** [comment]
Anybody have a fix for this. Same thing happens for me.
